docs/requirements.in

```
@@ -1 +1,2 @@
 Sphinx
+sphinx_ansible_theme
```

as per earlier suggestion
```suggestion
sphinx_ansible_theme
..
```

---

Commented above. We can try this approach after enabling dependabot, I want to be sure it works well with it too.